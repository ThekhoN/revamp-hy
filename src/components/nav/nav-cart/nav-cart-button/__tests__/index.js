import React from 'react';
import renderer from 'react-test-renderer';
import NavCartButton from '../index';

describe('NavCartButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <NavCartButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
