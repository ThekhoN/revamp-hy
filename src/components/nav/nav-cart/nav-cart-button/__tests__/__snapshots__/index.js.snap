// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavCartButton renders correctly 1`] = `
<div
  className="nav-cart-button"
  onClick={undefined}
>
  <span
    className="svg-icon-wrapper"
  >
    <svg
      aria-labelledby="title"
      height="48"
      viewBox="0 0 48 48"
      width="48"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title
        id="title"
      >
        Homeyantra Navigation Close
      </title>
      <path
        d="M12 15.5c-.7 0-1.1.9-.7 1.5.3.4.7.4 1.1.4.8.1 1.7.1 2.5.1.4 1.5.8 2.9 1.2 4.4.5 1.8.8 3.8 1.6 5.5.3.6.8.6 1.4.6h6.2c1.8 0 3.8.1 5.6 0 .7-.1 1-.4 1.2-1 .4-.9.7-1.9 1.1-2.8.4-1.1.8-2.1 1.2-3.2.3-.8.7-1.9-.6-2-1.2-.2-2.6 0-3.8 0H17.2c-.3-1-.4-3.2-1.6-3.4-1.1-.2-2.4-.2-3.6-.1zm6.1 5.5h14.2l-2.1 5H19.5c-.4-1.7-.9-3.3-1.4-5zm2.8 7.6c-3.3 0-3.2 5.1 0 5.1 3.2-.1 3.2-5.1 0-5.1zm7.6 0c-3.3 0-3.2 5.1 0 5.1 3.2-.1 3.3-5.1 0-5.1zm-7.6 1.8c.9 0 .9 1.4 0 1.4s-.9-1.4 0-1.4zm7.6 0c.9 0 .9 1.4 0 1.4s-.9-1.4 0-1.4z"
        fill={undefined}
      />
    </svg>
  </span>
</div>
`;
